dts/bindings/modem/quectel,bg9x.yaml

```diff
@@ -10,7 +10,6 @@ include: uart-device.yaml
 properties:
   mdm-power-gpios:
     type: phandle-array
-    required: true
```

Comment:
If removing a validation from here, please make sure consumers (the driver) check for this property e.g. using assertions